src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -2226,7 +2226,7 @@ namespace EvaporativeFluidCoolers {
                 // FanModeFrac = 1.0;
                 this->FanPower = FanPowerOn;
             }
-        } else if (inletWaterTemp <= TempSetPoint) {
+        } else {
```

Comment:
[src/EnergyPlus/EvaporativeFluidCoolers.cc:2229]:(style),[multiCondition],Expression is always true because 'else if' condition is opposite to previous condition at line 2204.